eth/downloader/queue.go

```diff
@@ -70,14 +70,14 @@ type fetchResult struct {
 	Receipts     types.Receipts
 }
 
-func newFetchResult(header *types.Header, fastSync bool) *fetchResult {
+func newFetchResult(header *types.Header) *fetchResult {
 	item := &fetchResult{
 		Header: header,
 	}
 	if !header.EmptyBody() {
 		item.pending |= (1 << bodyType)
 	}
-	if fastSync && !header.EmptyReceipts() {
```

Comment:
I don't think this is right. Since `fastSync` is gone, this block should also be gone

Reply:
Removed